.github/workflows/backport.yml

```diff
@@ -3,6 +3,11 @@ on:
   issue_comment:
     types: [created]
 
+permissions:
+  contents: write
+  issues: read
+  pull-requests: write
```

Comment:
@hoyosjs I get `pull-requests: write` but why is `write` needed for `contents` or `read` for `issues`❔

Reply:
I expected

``` yaml
permissions:
  contents: read
  pull-requests: write

Reply:
All comments fall under "issues" in the GitHub API IIRC, even PR comments, so that's why `read` is required for `issues`.

`write` is probably required for `contents` since this workflow creates a new branch and commits as part of how it functions.

Reply:
`pull-requests: write` is only for the explicit action to create the pull request from one branch to another.

Reply:
Yes, exactly those scopes are why I gave it those permissions. @vcsjones do you know if this is correct?

Reply:
You might need `write` on `issues` because the backport bot leaves a comment on PRs ([like these](https://github.com/dotnet/runtime/pull/61590#issuecomment-969346934)) that the back port has been started. @jkoritzinsky is right, PR comments are "issues" (also evidence of this that this backport bot triggers on `issue_comment`.

Reply:
I forgot about the reply. Much appreciated @vcsjones